src/apiflask/app.py

```diff
@@ -16,7 +16,11 @@
 from flask import jsonify
 from flask import render_template_string
 from flask.config import ConfigAttribute
-from flask.globals import _request_ctx_stack
+try:
+    from flask.globals import request_ctx
+except ImportError:
+    from flask.globals import _request_ctx_stack
+    request_ctx = None
```

Comment:
Imo we should ignore the type here.

```suggestion
    request_ctx = None  # type: ignore
```